src/CHANGES.txt

```diff
@@ -177,7 +177,8 @@ RELEASE 3.1.0.alpha.yyyymmdd - NEW DATE WILL BE INSERTED HERE
     - Typo in customized decider example in user guide
     - Replace usage of unittest.TestSuite with unittest.main() (fix #3113)
 
-
+  From Arda Fu
+    - Fix cpp regex error that match 'if' in 'ifndef'.
```

Comment:
Change to be "Fix cpp scanner regex logic to treat ifndef. Previously it was not properly differentiating between if, ifdef, and ifndef"
Assuming this is accurate and I'm understanding this change..